pkg/istio-agent/agent.go

```diff
@@ -216,11 +216,11 @@ type AgentOptions struct {
 // health checking for VMs and DNS proxying).
 func NewAgent(proxyConfig *mesh.ProxyConfig, agentOpts *AgentOptions, sopts *security.Options, eopts envoy.ProxyConfig) *Agent {
 	return &Agent{
-		proxyConfig:   proxyConfig,
-		cfg:           agentOpts,
-		secOpts:       sopts,
-		envoyOpts:     eopts,
-		caFileWatcher: filewatcher.NewWatcher(),
+		proxyConfig: proxyConfig,
+		cfg:         agentOpts,
+		secOpts:     sopts,
+		envoyOpts:   eopts,
+		fileWatcher: filewatcher.NewWatcher(),
```

Comment:
do we need the filewatcher changes now?

Reply:
Looks like its just generic now so not bad

Reply:
yes